RetailCoder.VBE/Inspections/Results/AssignedByValParameterInspectionResult.cs

```diff
@@ -25,9 +26,10 @@ public override IEnumerable<QuickFixBase> QuickFixes
         {
             get
             {
+                IAssignedByValParameterQuickFixDialogFactory factory = new AssignedByValParameterQuickFixDialogFactory();
```

Comment:
So, do you mean moving this (or a similar) line of code to the constructor of the Inspection class?  I'm OK with moving it....I'm wondering if the constructor of the inspection should carry the factory interface dependency for a dialog component of one of the three quickfixes - it seems  a little 'high up' - but it's likely I'm not understanding what you are advocating.  Could you comment a bit more?  Thanks.